components/authz-service/storage/v2/storage.go

```diff
@@ -254,6 +255,7 @@ func DefaultRoles() []Role {
 			"event:*:list",
 			"ingest:*:get",
 			"ingest:*:list",
+			"applications:*:list",
```

Comment:
is there a possibility there will be an individual `getService` in the near future? if so we may want to add `applications:*:get` as well while we're here